src/hammer-tech/schema.json

```diff
@@ -194,7 +194,64 @@
             "type" : "string"
           }
         },
-        "additionalProperties" : false
+        "additionalproperties" : false
+      }
+    },
+    "stackups" : {
+      "type" : "array",
+      "items" : {
+        "title" : "Stackup",
+        "type" : "object",
+        "additionalproperties" : false,
```

Comment:
What does this do?

Reply:
@edwardcwang ? This is in all the other objects. I assume it disallows adding additional properties not specified in the schema.